### ISSUE TITLE
Roadmap for scss files

### ISSUE THREAD

**Issue by crftwrk** [bootscore/bootscore#47]
Since we have source in theme, I'm racking my brain how to merge them all to one single css at the end without disturbing users too much. I'd like to share some thoughts about a clear roadmap.

### 5.1.3.1

- [x] Move content from `style.css` to scss files. Done by https://github.com/bootscore/bootscore/pull/33
- [x] Add a new empty `_bscore_woocommerce.scss` to collect WooCommerce styles. https://github.com/bootscore/bootscore/pull/41
- [x] Inform bootcommerce user that they should **add** `@import "bscore_woocommerce";` to their child https://bootscore.me/blog/bootstrap/bootscore-5-1-3-1/
- [x] Inform users that they should **remove** `@import "fontawesome";` in their child https://bootscore.me/blog/bootstrap/bootscore-5-1-3-1/

### 5.2.0.0

- [x] Because users added `@import "bscore_woocommerce";`, we are able to move content from `woocommerce-style.css` to scss, delete file and remove enqueue snippet in `woocommerce-functions.php`. One file request less. https://github.com/bootscore/bootscore/pull/82
- [x] Because users removed `@import "fontawesome";` in child, we can delete this file. We should move content from `fontawesome.min.css` to somewhere in the subfolder `scss/bootscore/`. We can delete file and remove enqueue snippet in functions.php. One file request less. By doing this, we have more control to handle icon-font or inline-svg. Discussed here https://github.com/bootscore/bootscore/issues/46. https://github.com/bootscore/bootscore/pull/160
- [x] Adding a source map https://github.com/bootscore/bootscore/pull/98.

### 5.2.X.X

This is the hardest step, users have to edit enqueue script in child.

- [x] ~~At best, we set compiled output to `style.css`. So there is just one minifed css file at the end. Best performance, but hard to handle in child. https://github.com/bootscore/bootscore/tree/compiler-to-style.css~~
- [x] Better to handle in child. We can rename `_bootstrap.min.scss` to `main.css` or whatever. Because file contains now all css, name `bootstrap.min.css` is not correct anymore and confusing. https://github.com/bootscore/bootscore/pull/99

In conclusion, first collect all css to scss, then move the final file to somewhere else. Similar breaking changes in the past have shown that users will accept them if this serves the purpose and is logical. Think we are addressing around 400 - 600 active maintainers here.

@justinkruit @Axos11 any better suggestions?
